lang/en.json

```diff
@@ -1817,6 +1817,25 @@
 "DND5E.WeaponSimpleProficiency": "Simple",
 "DND5E.WeaponSimpleR": "Simple Ranged",
 "DND5E.Weight": "Weight",
+"DND5E.WeightUnit": {
+  "Label": "Weight Units",
+  "Kilograms": {
+    "Label": "Kilograms",
+    "Abbreviation": "kg"
+  },
+  "Megagrams": {
+    "Label": "Tonnes",
```

Comment:
What's wrong with "Megagram" and "mg"?

Reply:
I thought `Tonnes` was the more generally used term, but I'd welcome feedback from someone in a metric country :)

Reply:
I am, lol, but here's another to make a fool of me.